packages/flutter_tools/lib/src/android/gradle_utils.dart

```diff
@@ -17,41 +20,46 @@ import '../cache.dart';
 import '../globals.dart' as globals;
 import '../project.dart';
 import '../reporting/reporting.dart';
-import 'android_studio.dart';
 
-/// The environment variables needed to run Gradle.
-Map<String, String> get gradleEnvironment {
-  final Map<String, String> environment = Map<String, String>.of(globals.platform.environment);
-  if (javaPath != null) {
-    // Use java bundled with Android Studio.
-    environment['JAVA_HOME'] = javaPath;
-  }
-  // Don't log analytics for downstream Flutter commands.
-  // e.g. `flutter build bundle`.
-  environment['FLUTTER_SUPPRESS_ANALYTICS'] = 'true';
```

Comment:
We don't care about this anymore?

Reply:
No, we specifically don't want it actually - this is a holdover from each build requiring N flutter commands